selectrum.el

```diff
@@ -286,22 +287,28 @@ new one."
       (setcar lst (funcall func (car lst)))
       (setq lst (cdr lst)))))
 
-(defun selectrum--move-to-front-destructive (elt lst)
+(defun selectrum--move-to-front-destructive (elt lst &optional add)
   "Move all instances of ELT to front of LST, if present.
-Make comparisons using `equal'. Modify the input list
-destructively and return the modified list."
+If ADD is non-nil add element if not present. Make comparisons
+using `equal'. Modify the input list destructively and return the
+modified list."
```

Comment:
This should specify where in the list the item is added (at the front or at the end?).

Reply:
Done